### PR TITLE
Improve description of module frontend editing setting in com_config

### PR DESCRIPTION
**Pull request by Hackwar**
See title.

### CODE REVIEW

**Comment by wilsonge** [comment]
:+1: although I'd suggest

```this has a performance penalty for logged in users``` rather than the ```significant performance penalty```

**Comment by Hackwar** [comment]
There you go

**Comment by wilsonge** [comment]
significant has a typo :stuck_out_tongue_winking_eye:

**Comment by Hackwar** [comment]
fixed.

**Comment by Kubik-Rubik** [comment]
@wilsonge Eagle eyes! :-)

@Hackwar Good improvement, thank you!

**Comment by brianteeman** [comment]
I thought we were in language freeze - another PR that changes a single character has already been rejected :(

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 1]
```diff
@@ -73,8 +73,8 @@ COM_CONFIG_FIELD_FILTERS_CUSTOM_BLACK_LIST="Custom Black List"
 COM_CONFIG_FIELD_FILTERS_NO_HTML="No HTML"
 COM_CONFIG_FIELD_FILTERS_NO_FILTER="No Filtering"
 COM_CONFIG_FIELD_FILTERS_WHITE_LIST="White List"
-COM_CONFIG_FRONTEDITING_DESC="Select if you want mouse-over edit icons for modules and menu items (support may depend on your template)."
-COM_CONFIG_FRONTEDITING_LABEL="Mouse-over Edit Icons for"
+COM_CONFIG_FRONTEDITING_DESC="This setting allows to display mouse-over edit icons for modules and menu items (support may depend on your template). Please notice that this has a significant performance penalty for logged in users."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: "this setting allows to display"
This is nonsense english

**Comment by wilsonge** [comment]
and brian if you notice i set the milestone to 3.4.1 ;)

File: administrator/language/en-GB/en-GB.com_config.ini (hunk 1 above)
Comment (Hackwar) [inline administrator/language/en-GB/en-GB.com_config.ini]: Would it be possible for you to correct this? Me english not prefect. :wink: As you properly noted, our english language strings are sub-optimal and I would rather have a native speaker propose something better, maybe as a direct PR against my PR, than me doing 30 iterations to get this right.

**Comment by Hackwar** [comment]
Brian, I'm arguing with Thomas about this feature in #6041 and the argument right now is, that the switch is so hard to find, that we can't expect people to switch this on again if we were to switch it off and thus we can't change that setting. So since nobody seems to want to change this language string, I went ahead and created a PR. I'm very happy if someone else would propose something better, since, while I call my English good, I'm very far from perfect. But I also don't want to let anybody say "hey, its difficult to find, nobody is doing something against that, so we can't change this."

**Comment by wilsonge** [comment]
RTC

**Comment by brianteeman** [comment]
George the english is terrible and makes no sense surely even you can see that

File: administrator/language/en-GB/en-GB.com_config.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: This setting will display mouse...

**Comment by wilsonge** [comment]
Maybe I shouldn't be doing this at this time of night.....

**Comment by wilsonge** [comment]
Can you please do a replacement PR or PR into Hannes branch please. Because Hannes is right as well that what's there equally makes no sense

**Comment by brianteeman** [comment]
But is it even correct to say it has a significant performance hit. There is an effect but isnt that a bit strong?

**Comment by wilsonge** [comment]
Well this was the cause of that 33% hit we saw in 3.4. We only fixed it for users not logged in. I dunno how hard we wanna be with the wording but we probably should let the users know there is a penalty for using it.

**Comment by brianteeman** [comment]
I'm agreeing with the need for a message but if we make it that hard no one
will ever use it and yet we are stating it is a key feature of the release.
I would prefer a softer notice and then make it a code priority to look at
resolving the performance issue. Otherwise we should drop the feature
completely

**Comment by zero-24** [comment]
Setting to needs review until someone give the advice to use this text or another. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6105">issues.joomla.org/joomla-cms/6105</a>.</sub>

File: administrator/language/en-GB/en-GB.com_config.ini (hunk 1 above)
Comment (roland-d) [inline administrator/language/en-GB/en-GB.com_config.ini]: What if we change the text `Please notice that this has a significant performance penalty for logged in users.` to `The user experience for logged-in users may be sub-optimal when enabling this option`?

**Pull request closed without merging**